bzr.go

```diff
@@ -146,6 +147,40 @@ func (s *BzrRepo) Tags() ([]string, error) {
 	return tags, nil
 }
 
+// CurrentVersionsWithRevs returns a list of available branches and tags, and
+// their underlying revision identifiers. The list is guaranteed to be current.
+//
+// As the model we use for bzr interaction (and go get generally does as well)
+// involves interacting only with a bzr repository's default branch, we return
+// only tags here.
+func (s *BzrRepo) CurrentVersionsWithRevs() (versions []VersionInfo, localSynced bool, err error) {
+	var out []byte
+	// Update the local first
+	_, err = s.runFromDir("bzr", "pull")
+	if err != nil {
+		return
+	}
+	localSynced = true
```

Comment:
yup, will do both today

On 4/13/16 09:40, Matt Farina wrote:
> In bzr.go <https://github.com/Masterminds/vcs/pull/21#discussion_r59548772>:
> 
>> @@ -146,6 +147,40 @@ func (s *BzrRepo) Tags() ([]string, error) {
>>  	return tags, nil
>>  }
>>  
>> +// CurrentVersionsWithRevs returns a list of available branches and tags, and
>> +// their underlying revision identifiers. The list is guaranteed to be current.
>> +//
>> +// As the model we use for bzr interaction (and go get generally does as well)
>> +// involves interacting only with a bzr repository's default branch, we return
>> +// only tags here.
>> +func (s *BzrRepo) CurrentVersionsWithRevs() (versions []VersionInfo, localSynced bool, err error) {
>> +	var out []byte
>> +	// Update the local first
>> +	_, err = s.runFromDir("bzr", "pull")
>> +	if err != nil {
>> +		return
>> +	}
>> +	localSynced = true
> 
> @sdboyer <https://github.com/sdboyer> This now needs to merge in master
> to get the |RunFromDir| change. Can you setup |IsSynced()| as you suggest?
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Masterminds/vcs/pull/21/files/f2e4eeaab278a5130c545d0142e6517687e8aacc#r59548772>
>